### ISSUE TITLE
Blade Format breaking line and following directives

### ISSUE THREAD

**Issue by Pymous** [onecentlin/laravel-blade-snippets-vscode#78]
Hello ! 

First of all, thanks for your extension, it's really nice !

I was discussing about an issue on the Larachat Slack, about Blade Formatting breaking line, and I managed to reproduce the issue with a minimum of code.

```php
<div>
    <div>
        <div>
            <div>
                <div>
                    <!-- This line will format "properly" even if inlining everything is a bit messy -->
                    <input id="user.telephone_secondary" value="{{ old('user.telephone_secondary') }}" class="form-control we need to have a very long list of classes i think its related to the line of the line and when we go past a threshold it will break">
                    @if ($errors->has('user.telephone_secondary')) Anything @endif
                    <!-- This line will not format properly and throw an exception -->
                    <input class="form-control we need to have a very long list of classes i think its related to the line of the line and when we go past a threshold it will break" id="user.telephone_secondary" value="{{ old('user.telephone_secondary') }}">
                    @if ($errors->has('user.telephone_secondary')) Anything @endif
                </div>
            </div>
        </div>
    </div>
</div>
```

Formating this template will format the code to look like that and throw a syntax exception :
![image](https://user-images.githubusercontent.com/13593371/44994162-a5baa900-af9d-11e8-95c0-e24dbeea5f54.png)

I'm pretty sure it's related to the length of the previous line, and if the formatter can break the line or not. If it can, it will, then it will inline the next directive and line break it too.

Maybe adding a line break before every directives instead of inlining them would fix the issue ? 

Hope you can do something about it, and thanks again for your work !

**Comment by onecentlin** [comment]
Fixed in version 1.20.0, please update and try again, thanks!

**Comment by MarcinKilarski** [comment]
Hi,

Unfortunately, I have version 1.20.0 and it still breaks lines causing errors.

Example 1
`<section id="section-number-{{ $sectionCount }}" class="<section-number-{{ $sectionCount }} s-{{ $section['acf_fc_layout'] }}">`

`<section id="section-number-{{ $sectionCount }}"
  class="<section-number-{{ $sectionCount }} s-{{ $section['acf_fc_layout'] }}">`

Example 2
`      $image_alt = $card['alt']; // Option: $card['image_alt_tag'] = use custom alt tag from the ACF; comment out this line or set value to '' to use the default image alt tag defined in the Media library`

`      $image_alt = $card['alt']; // Option: $card['image_alt_tag'] = use custom alt tag from the ACF; comment out this
      line or set value to '' to use the default image alt tag defined in the Media library`

Do you need to make any changed in the Setting to prevent it from splitting long lines?
